hyperv/datadog_checks/hyperv/data/conf.yaml.example

```diff
@@ -1,4 +1,12 @@
 init_config:
 
 instances:
-  - {}
```

Comment:
Not sure we want to get rid of the `{}`

Reply:
I thought the same but config validation passes 🤷‍♂️

Reply:
and does it work with an agent ?

Reply:
@ofek was able to test this locally with success 🎉